### PR TITLE
Documentation for CR

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

**Comment by JStickler** [comment]
Some of this is already documented in PR#14.  Could that PR be merged and then updated?

**Comment by brian-avery** [comment]
@JStickler I'm only seeing the examples there. Let me check with Bill, but I think we need something explaining what the arguments mean. Do you think this content should all be in the same page as the install doc? Don't know how long the CR will eventually be.

**Comment by JStickler** [comment]
If this is going to be a separate topic with specifics about the arguments in a custom resource, there should at least be a link to this page from the installation page that gives the cr examples.  Otherwise users won't know where to find the details.

(Also that PR just needs to merge at some point....)

File: topics/istiodocs/custom-install.adoc [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+|maistra if deployment_type is origin and openshift-istio-tech-preview if deployment_type is openshift 
+
+
+|version
+|docker tag to use with Jaeger images
+|valid docker tag
+|v0.5.0
+
+|elasticsearch_memory
+|The amount of memory to allocate to the elastic search install
```
Comment (JStickler) [inline topics/istiodocs/custom-install.adoc]: This isn't showing up in the display because you've only specified three columns (You need another | )

Comment (brian-avery) [reply topics/istiodocs/custom-install.adoc]: Thanks @JStickler. Fixed.

**Pull request merged**